app/serializers/user_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class UserSerializer < ActiveModel::Serializer
+  attributes :id, :first_name, :last_name, :email, :rents_count
+
+  def rents_count
+    Rent.where('user_id = ?', object.id).count
```

Comment:
Not here! You should't do queries in a Serializer, their only propose is adapt how the data is displayed and not generate/retrieve it. You can do something more "transparent" as `object.rents.count`